lib/nexmo/oas/renderer/public/assets/javascripts/nexmo-oas-renderer.js

```diff
@@ -1,61 +1,64 @@
-if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
-  var h = window.location.href;
-  if (h.indexOf("?theme=") === -1) {
-    window.location.href = h + "?theme=dark";
+if(document.querySelector(".oas-wrapper")){
+  if (window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches) {
+    var h = window.location.href;
+    if (h.indexOf("?theme=") === -1) {
+      window.location.href = h + "?theme=dark";
```

Comment:
@mheap can we also change this to something like :
```
    var queryString = new URLSearchParams(window.location.search);
    queryString.set("theme", "dark");
    window.location.search = queryString.toString();
```
that way we fix the issue of having multiple `?` in the url